tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,15 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel
+
+require('babel-core/register')(require('./babel-config.js')); // #RemoveInProd this line is removed in isopack.js
```

Comment:
Looking into my crystal ball, I think we will end up writing [our own version of `require("babel/register")`](https://github.com/meteor/babel/blob/master/register.js), where we can install the stack trace rewriter as correctly as we want.

I know I asked for separate source map files earlier in this thread, but if the stack trace rewriter only needs inline maps, I would not be opposed to using only inline source maps. That's what [`node-inspector`](https://www.npmjs.com/package/node-inspector) seems to handle best, as well.